### ISSUE TITLE
Library pagination page numbers wrapping unnecessarily 

### ISSUE THREAD

**Issue by peichins** [QutBioacoustics/baw-client#311]
Page number buttons on the annotation library page are wrapping even though there is plenty of space for them. This happens when the browser window is under about 1600 pixels wide. 

https://www.ecosounds.org/library

![image](https://cloud.githubusercontent.com/assets/5986336/25213029/5c79d6c8-25d1-11e7-89db-a6a8d7cf9dd3.png)

**Comment by atruskie** [comment]
Yeah, I confirm this. It's something to do with bootstrap's grid layout system. Feel free to have a stab at it.
